.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -16,16 +16,16 @@ A clear and concise description of what you expected to happen.
 
 One or more links to the documentation page, preferably to the according
 section. If applicable, link also to other resources which might be helpful
-to process your bug report.
+to process your report.
 
 ### TYPO3 versions
 
 [e.g. 11, 12]
 
 ### Possible Solution
 
-Only if you have suggestions on a fix for the bug.
+Only if you have concrete suggestions to solve the problem.
 
 ### Additional context
 
-Add any other context or screenshots about the feature request here.
+Add any other context or screenshots about the request here.
```

Comment:
Add any other context or screenshots about your proposition here.

Reply:
Chose "proposal" instead of "proposition"